frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/components/editBasicInfo.jspf

```diff
@@ -50,8 +50,8 @@
     <tr>
         <td>
             <div class="form-group">
-                <label for="comp_componenttype">Component Type</label>
-                <select class="form-control" id="comp_componenttype" name="<portlet:namespace/><%=Component._Fields.COMPONENT_TYPE%>">
+                <label class="mandatory" for="comp_componenttype">Component Type</label>
+                <select class="form-control" id="comp_componenttype" required name="<portlet:namespace/><%=Component._Fields.COMPONENT_TYPE%>">
```

Comment:
Please add a appropriate error message for mandatory field component type.

Reply:
Done.